src/core/crypto/ecdsa.hpp

```diff
@@ -164,6 +171,12 @@ class P256
          */
         uint8_t GetDerLength(void) const { return mDerLength; }
 
+        /**
+         * This method clears the keypair stored.
+         *
+         */
+        void Clear(void) { memset(mDerBytes, 0, kMaxDerSize); }
```

Comment:
If we use `Settings::Delete<Key>`  then we won't need this

Reply:
Will push a fix in next commit.